eclipse-scout-core/src/tree/Tree.ts

```diff
@@ -2408,6 +2408,15 @@ export class Tree extends Widget implements TreeModel {
       updatedNodes.push(oldNode);
     });
 
+    // recompute node size if horizontal scrolling is enabled
+    if (this.isHorizontalScrollingEnabled()) {
```

Comment:
I remembered again that updateNodes is probably not the right method to adjust. Unfortunately, there is another method with a similar purpose: changeNode. changeNode should be called if a property changes that affects presentation but not the tree structure (see TreeEvent.TYPE_NODE_CHANGED). So we should add it there (to make it work for Scout Classic as well), but it works a bit differently... Unfortunately, it is not a batch method, instead it uses a microTask to coalesce the update. I tried to implement it here: https://github.com/eclipse-scout/scout.rt/tree/features/cgu/25.1/tree_update_node_size_402635 But it needs some more work.

I think in the future we should combine updateNodes and changeNode because it is confusing.